x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -285,23 +282,23 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('be.visible');
   });
 
-  it('Changed banner should hide on saving changes', () => {
+  it('should hide Changed banner on saving changes', () => {
     visitAlertsPageWithCustomFilters(customFilters);
     waitForPageFilters();
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('be.visible');
     saveFilterGroupControls();
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('not.exist');
   });
 
-  it('Changed banner should hide on discarding changes', () => {
+  it('should hide Changed banner on discarding changes', () => {
     visitAlertsPageWithCustomFilters(customFilters);
     waitForPageFilters();
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('be.visible');
     discardFilterGroupControls();
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('not.exist');
   });
 
-  it('Changed banner should hide on Reset', () => {
+  it('should hide Changed banner on Reset', () => {
```

Comment:
we're using `cy.get(ALERTS_REFRESH_BTN).trigger('click');` line 323 and 348, should we extract that into a `task` function?

Reply:
also we should probably use `click` instead of `trigger('click')` right?

Reply:
> also we should probably use click instead of trigger('click') right?

done

>we're using cy.get(ALERTS_REFRESH_BTN).trigger('click'); line 323 and 348, should we extract that into a task function?

There was already a function so I removed ALERTS_REFRESH_BTN and replace it with task